uiowa/uiowa
# Use loggerChannelTrait again

Resolves #7597 





# How to test




The following sitenow and site drush commands have been updated:

- Uiowa Core: `uiowa_core:toggle-gtag`, `uiowa_core:custom_role_mappings`
- Sitenow p2lb: `sitenow_p2lb:cleanup`
- its.uiowa.edu: `its_core:alerts-digest`
- brand.uiowa.edu: `brand_core:lockup-digest`
- classrooms.uiowa.edu: `classrooms_core:buildings`, `classrooms_core:rooms_import`
- facilities.uiowa.edu: `facilities_core:buildings_import`, `facilities_core:projects_import`

For each, test that on a fresh site sync 

1. the command functions and you see terminal output
2. Enable dblog and re-run the command, and check that
   1. you see terminal output 
   2. watchdog has caught the log (eg `ddev drush @site.local watchdog-show --count=10` you see a line item for what you just ran, or at /admin/reports/dblog)
3. Clear cache on the site you're working with, re-run again, and check that 
   1. you see terminal output
   2. watchdog has caught the log

If the command ran properly in step 1, for steps 2 and 3 you only need to let it run far enough to get output and/or logging (for instance, facilities building import, you can cancel after the initial "starting" message)
